trade-finance-manager-ui/server/controllers/helpers/can-deal-be-cancelled.helper.test.ts

```diff
@@ -0,0 +1,79 @@
+import { DEAL_SUBMISSION_TYPE, TEAM_IDS, TFM_DEAL_CANCELLATION_STATUS } from '@ukef/dtfs2-common';
+import { TfmSessionUser } from '../../types/tfm-session-user';
+import { canDealBeCancelled, canSubmissionTypeBeCancelled } from './can-deal-be-cancelled.helper';
+
+const pimUser = { teams: [TEAM_IDS.PIM] } as TfmSessionUser;
+const nonPimUser = { teams: [TEAM_IDS.UNDERWRITERS] } as TfmSessionUser;
+
+const { AIN, MIN, MIA } = DEAL_SUBMISSION_TYPE;
+const { COMPLETED, SCHEDULED, DRAFT } = TFM_DEAL_CANCELLATION_STATUS;
+
+describe('canDealBeCancelled', () => {
+  describe('when `FF_TFM_FACILITY_END_DATE_ENABLED` is set to false', () => {
+    describe('when user is a PIM user', () => {
+      it.each([MIA, MIN, AIN])('should return false if the deal type is %s and the cancellation status is completed', (type) => {
+        const result = canDealBeCancelled(type, pimUser, COMPLETED);
```

Comment:
could these `it.each`'s be `describe.each`?

```suggestion
      describe.each([MIA, MIN, AIN])('when the deal type is %s', (type) => {
        it('should return false if the cancellation status is completed', () => {
          const result = canDealBeCancelled(type, pimUser, COMPLETED);

          expect(result).toEqual(false);
        });

        it('should return false if the cancellation status is scheduled', (type) => {
          const result = canDealBeCancelled(type, pimUser, SCHEDULED);

          expect(result).toEqual(false);
        });

        // etc...
      });
```

Reply:
actually is this just copied over & git being weird? feel free to ignore

Reply:
Good spot, that is a valid and better change - have updated the test files. Thanks!